contracts/crowdsale/validation/TimedCrowdsale.sol

```diff
@@ -13,6 +13,13 @@ contract TimedCrowdsale is Crowdsale {
     uint256 private _openingTime;
     uint256 private _closingTime;
 
+    /**
+     * Event for crowdsale extending
+     * @param newClosingTime new closing time
+     * @param prevClosingTime old closing time
+     */
+    event TimedCrowdsaleExtended(uint256 newClosingTime, uint256 prevClosingTime);
```

Comment:
Please reverse the order of the two arguments here for consistency with the rest of OpenZeppelin. `Owable` for example places the previous owner first and the new one second.

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/fc17a1d9f58b7ca6e2de884769f8b6b362dc0e3c/contracts/ownership/Ownable.sol#L11

Also for consistency we should be using the full word `previous` instead of `prev`.